views/const.go

```diff
@@ -3,16 +3,24 @@ package views
 const (
 	templatesFolder     string = "templates"
 	TemplateCSS         string = "../static/styles/template.css"
-	LoginCSS            string = "../static/styles/login-style.css"
+	LoginCSS            string = "../static/styles/style.css"
```

Comment:
Good decision with the name!

Reply:
rename it coz this file applies to every page
prob shoud rename LoginCSS to something like template ?